DB/src/main/java/io/deephaven/db/tables/ColumnDefinition.java

```diff
@@ -480,118 +479,167 @@ public boolean isFixedWidthObjectType() {
 
     // TODO: DELETE THESE OVERRIDES (SEE NOTES IN TableDefinition)
```

Comment:
You can and should delete this TODO now, as well as `MAGIC_NUMBER` (which has some crappy test use case that can be served some other way).